docs/usage/configuration-options.md

```diff
@@ -681,6 +681,9 @@ Example:
 
 ## gitLabAutomerge
 
+If you enabled automerge in the Renovate config, you can speed up the automerge process, by using GitLab's own automerge function.
```

Comment:
```suggestion
If you enabled `automerge` in the Renovate config, you can speed up the automerge process, by using GitLab's own automerge function.
```

so highlight `automerge` as an renovate option

Reply:
Doesn't Gitlab automerge speed to renovate automerge, not vice versa?